src/views/course_pairings.vue

```diff
@@ -203,7 +203,11 @@ export default {
             if (this.table != null) {
                 this.table.destroy();
             }
-            this.table = new Handsontable(container, this.tableSettings);
+
+            this.table =
+                container == null
+                    ? null
+                    : new Handsontable(container, this.tableSettings);
```

Comment:
`this.table = container || new Handsontable...`

Reply:
This format doesn't work. I'll just use the long format for the if statement.